test/adagrams.test.js

```diff
@@ -4,6 +4,7 @@ import {
   scoreWord,
   highestScoreFrom,
 } from "adagrams";
+import Adagrams from "../src/adagrams";
```

Comment:
Glad to see you got practice creating a class in JS! Since you import the Adagrams class, you can delete lines 1-6 since we no longer need to import individual methods and they're now unused. We'll always want to delete unused imports.